Gemfile

```diff
@@ -34,6 +35,8 @@ gem 'clam_scan'
 gem 'config'
 gem 'connect_vbms', git: 'https://github.com/department-of-veterans-affairs/connect_vbms.git', branch: 'master'
 gem 'date_validator'
+gem 'dry-struct'
+gem 'dry-types'
```

Comment:
I think we also should include 'fast_jsonapi' here in `Gemfile` too.
If the user doesn't have it already installed, this error occurs:
<img width="747" alt="Screen Shot 2019-07-11 at 3 44 41 PM" src="https://user-images.githubusercontent.com/733873/61080475-271b6d00-a3f3-11e9-85b9-a866c96081bc.png">

Reply:
👍weird it's defined in the gemspec file, I would've thought it gets installed when the bundle runs, but I'll add it in to the main Gemfile